_docs/persistenstorage-provider.md

```diff
@@ -0,0 +1,103 @@
+# Lets begin
+
+At Akash we use Rook/Ceph couple to provider dynamic volumes provisioning
+Ceph - distributed filesystem
+Rook - Kubernetes operator to control/provision Ceph
+
+In the guide **Rook** means Rook/Ceph couple.
+This guide is focused on Rook setup and usage for Akash provider. To get familiar with Rook concepts refer to the [official documentation](https://rook.github.io/docs/rook/v1.8/)
+Configuration is based on Rook V1.8
+
+## Prerequisites
+1. Helm charts referenced in the guide are present [here](https://github.com/ovrclk/helm-charts).
+2. Deploy and teardown are done by using bash script. Work on the helm chart is being done. Up until helm chart is ready this guide will be focused on creating test cluster only         
+Meanwhile examples referenced in the guide can be found [here](TBD)
+3. We will prevent cluster from using master nodes.
+4. We will be utilizing all available devices on each used node
+
+## Configuring
+In this example we are focusing on making slight adjustments to the `cluster.yaml` only. All other files must remain unchanged.
+
+1. get list of nodes in the cluster
+```shell
+kubectl get nodes -ojson | jq -r '.items[].metadata.labels."kubernetes.io/hostname"'
+k8s-master.edgenet-1.ewr1 # this node won't be used
```

Comment:
We should explain why this node is not used, not just say it isn't used. For example some people may run multi master kube clusters and would want to exclude all masters.